packages/twenty-front/src/modules/workflow/hooks/useGetUpdatableWorkflowVersion.ts

```diff
@@ -12,19 +12,25 @@ export const useGetUpdatableWorkflowVersion = () => {
   );
   const workflow = useWorkflowWithCurrentVersion(workflowVisualizerWorkflowId);
 
-  const getUpdatableWorkflowVersion = async () => {
+  const getUpdatableWorkflowVersion = async (): Promise<string> => {
```

Comment:
I think we like to suffix functions that throw with `orThrow`, like `getUpdatableWorkflowVersionOrThrow`